packages/esm-bed-management-app/src/routes.json

```
@@ -71,5 +71,13 @@
       "name": "new-bed-type-modal",
       "component": "newBedTypeModal"
     }
+  ],
+   "workspaces": [
+    {
+        "name": "add-edit-bed-workspace",
+        "title": "Add/Edit Bed",
```

```suggestion
        "title": "Add or edit bed",
```

I know this fallback gets overridden by whatever we pass along when we invoke `launchWorkspace`, but this seems more appropriate based on what the form does.